src/site/content/en/handbook/content-types/example-pattern/demo.njk

```diff
@@ -0,0 +1,56 @@
+---
+patternId: example-pattern
+---
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Sample demo</title>
+    <style>
+      html {
+        box-sizing: border-box;
+      }
+
+      *,
+      *:before,
+      *:after {
+        box-sizing: inherit;
+      }
+
+      body {
+        margin: 1rem;
+        font-family: system-ui, sans-serif;
+      }
+
+      code {
+        display: inline;
+        background: #f7f7f7;
+        padding: 4px;
+        border-radius: 8px;
+        line-break: anywhere
+      }
+    </style>
+  </head>
+  <body>
+    <h1>Sample demo</h1>
+
+    <img src="assets/fugu.svg" alt="Fugu fish." width="128" height="128">
```

Comment:
Can we rename `fugu.svg` to something that's non-specific? Maybe just `example.svg` - then it's clear this does not need to be updated if something about Fugu changes.